imports/api/logic/grad-checker/faculty/computing/computer-science/AY-1415/breadth-depth/focus-area/methods.js

```diff
@@ -18,25 +18,22 @@ export const checkFocusAreaFulfilmentMCs = function checkFocusAreaFulfilmentMCs(
   const focusArea4KKeys = studentFocusAreas.focusArea4KModules;
   const focusAreaPrimary4KKeys = studentFocusAreas.focusAreaPrimary4KModules;
   const focusAreaNonPrimaryKeys = studentFocusAreas.focusAreaNonPrimaryModules;
+  const modulesChecked = {};
   let totalMCs = 0;
   let fulfilmentRequirement = {
     requiredMCs: requiredMCs,
-    isFulfilled: false,
-    modulesChecked: {}    // swap this out to moduleChecked once focus area decision has been implemented
+    isFulfilled: false
   };
 
-  // for moduleChecked, only mark true if total number of MCs is less than or equal to
-  // the required number of MCs
-
   // for all primary focus area modules, check if module exists in student planner
```

Comment:
Might need to put each of the for loops into a separate function in the future so that their purpose and functionality can be documented and explained. For primary modules, 4k focus area modules, primary 4K focus area modules and non primary focus area modules